test/utils.js

```diff
@@ -145,7 +145,13 @@ async function runTest(name, expected, getResult, options) {
       } else if (typeof expected === "function") {
         expected(value);
       } else {
-        expect(value).toEqual(expected);
+        if (typeof value === "string") {
+          expect(value.replaceAll(result.fixturePath, "__FIXTURES__")).toEqual(
```

Comment:
this is me trying to account for the fact that the fixture path will be different on each run/machine

so we can't have it in snapshots or assertions

Reply:
moved this to its own pr in #51